packages/main/src/RPA/Smartsheet.py

```diff
@@ -0,0 +1,1415 @@
+        it is a valid but unknown row.
+        """
+        if isinstance(row, int):
+            if row not in [r.id for r in self.current_sheet.rows]:
+                return self._get_row_from_number(row)
+            else:
+                return row
+        elif isinstance(row, Row):
+            return row.id
+        else:
+            raise ValueError("Invalid row type.")
+
+    def _create_row_from_dict(self, row_dict: Dict) -> Row:
+        """Creates a row object from a dictionary."""
+        row = Row()
+        for key, value in row_dict.items():
+            cell = Cell()
+            if key in ["rowId", "row_id", "id"]:
+                row.id = value
+            elif key in ["rowNumber", "row_number"]:
+                row.id = self._get_row_from_number(value)
+            else:
+                cell.column_id = self._get_column_id(key)
+                cell.value = value
+                row.cells.append(cell)
+        return row
+
+    def _get_row_from_number(self, row_number: int) -> int:
+        """Returns the row ID from a row number. If not found, it
+        assumes the provided row_number may have been an ID and
+        returns it.
+        """
+        self._require_current_sheet()
+        for row in self.current_sheet.rows:
+            if row.row_number == row_number:
+                return row.id
+        return row_number
+
+    def _create_row_from_list(self, row_list: List[Dict]) -> Row:
+        """Creates a row object from a list."""
+        row = Row()
+        for cell_dict in row_list:
+            if cell_dict.get("title") == "rowId":
+                row.id = cell_dict.get("value")
+            elif cell_dict.get("title") == "rowNumber":
+                row.id = self._get_row_from_number(cell_dict.get("value"))
+            else:
+                cell = self._convert_dict_to_cell(cell_dict)
+                row.cells.append(cell)
+        return row
+
+    def _convert_data_to_row(self, data: Union[Dict, List, Row]) -> Row:
+        """Converts a dictionary or list to a row object."""
+        if isinstance(data, dict):
+            return self._create_row_from_dict(data)
+        elif isinstance(data, list):
+            return self._create_row_from_list(data)
+        elif isinstance(data, Row):
+            return data
+        else:
+            raise TypeError(
+                f"Invalid row type. Received {type(data)}, expected dict, list or Row."
+            )
+
+    @keyword
+    def convert_row_to_dict(self, row: Row) -> OrderedDict:
+        """Converts a row object to a dictionary."""
+        row_dict = OrderedDict()
+        for cell in row.cells:
+            row_dict[cell.column_id] = cell.value
+        return row_dict
+
+    @keyword
+    def get_row(
+        self,
+        row: Union[int, Row],
+        include: CommaListType = None,
+        exclude: CommaListType = None,
+        native: bool = False,
+    ) -> Row:
```

Comment:
```suggestion
    ) -> RowType:
```

Where `RowType = Union[OrderedDict, Row]` as such keyword can return both from what I see.